### ISSUE TITLE
commit_id field in status reports has value "(unkown)"

### ISSUE THREAD

**Issue by ndokos** [distributed-system-analysis/pbench#1624]
Server reports generated by the various server scripts are missing the commit_id field currently:

     @generated-by.commit_id	  	(unknown)

**Comment by portante** [comment]
@ndokos, perhaps we should let sleeping dogs lie, but found this script on our production node that we used in the past:

```
#! /usr/bin/env python3

import sys
configfile=sys.argv[1]

try:
    with open("/opt/pbench-server/VERSION") as f:
        version = f.read()[:-1]
except Exception:
    version=None

try:
    with open("/opt/pbench-server/SEQNO") as f:
        seqno = f.read()[:-1]
except Exception:
    seqno = None
try:
    with open("/opt/pbench-server/SHA1") as f:
        sha1 = f.read()[:-1]
except Exception:
    sha1 = None

if version is not None and seqno is not None and sha1 is not None:
    commit_id = "{}-{}{}".format(version, seqno, sha1)
    # back up the config file
    import shutil
    shutil.copy(configfile, "{}.orig".format(configfile))
    # and rewrite it, adding the commit_id. The rewrite
    # will delete comments and normalize the spacing around
    # delimiters as well.
    import configparser
    conf = configparser.ConfigParser()
    conf.read(configfile)
    conf.set('pbench-server', 'commit_id', commit_id)
    with open(configfile, "w") as f:
         conf.write(f)
```

**Comment by ndokos** [comment]
I can add the seqno to the ansible playbook easily enough. We can let the script go however - it's not worth keeping around. Agreed?

**Comment by ndokos** [comment]
Closed by PR #1625 and PR #1628.
